data/shaders/world_matrix.glsl

```diff
@@ -0,0 +1,7 @@
+mat4 world_matrix(vec4 i_World0, vec4 i_World1, vec4 i_World2) {
```

Comment:
I don't think it needs a header, tbh, since all you need is
`transpose(mat4(i_World0, i_World1, i_World2, vec4(0.0,0.0,0.0,1.0)))`

Reply:
Is transpose cheap operation?

Reply:
Yes.